Frameworks/UIKit/UIScrollView.mm

```diff
@@ -56,6 +58,7 @@
 static const bool DEBUG_DELEGATE = DEBUG_ALL || false;
 static const bool DEBUG_INSETS = DEBUG_ALL || false;
 static const bool DEBUG_ZOOM = DEBUG_ALL || false;
+static const bool DEBUG_DMANIP_GESTURE = DEBUG_ALL || true;
```

Comment:
yeah.

---
In reply to: [86598268](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86598268) [](ancestors = 86598268)